### PR TITLE
0.3.0 release

### PR DESCRIPTION
**Pull request by dhodovsk**
Please let me know if there is something forgotten in the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# 0.3.0
+
+## New Features
+
+* Introduced fixtures and helper functions that ease contributing
```
Comment (TomasTomecek) [inline CHANGELOG.md]: should be probably: *to simplify container testing*

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+* Support was added for another backend - nspawn
+* It is possible to decide cleanup policy when initializing backend.
+* Methods for listing containers and images were added to both supported backends
+* Image mounting is possible also without superuser privileges.
+* To represent and work with docker volumes, there is a new class `Volume`
+* `run_via_binary` accepts commands in form of list of strings
+  to make running images more intuitively.
+* It is possible to build image from Dockerfile using `DockerImage.build()`.
+* There is new method check signature hash of packages in image using
```
Comment (TomasTomecek) [inline CHANGELOG.md]: *...a new method to check GPG signatures of RPMs in images...*

**Comment by TomasTomecek** [comment]
Would be awesome if bot commented here what's he up to.

**Comment by phracek** [comment]
Version is still 0.2.0 https://github.com/user-cont/conu/blob/master/conu/version.py#L17 is it ok?

**Pull request merged**